vm/tests/solutions/spec/solr/index_page_spec.rb

```diff
@@ -15,11 +15,11 @@
 require 'spec_helper'
 
 describe command('curl -L -s -o /dev/null -w "%{http_code}" http://localhost/') do
-  its(:stdout) { should match /401/ }
+  its(:stdout) { should match /200/ }
```

Comment:
Why the response code is not 401 (Unauthorized)?

Reply:
I'm assuming with the [addition](https://issues.apache.org/jira/browse/SOLR-7896) of a login page to solr when a 401 is detected, we get served the login page instead. It's served to us as a 200. 

The tests would actually fail if we left this as expecting a 401.

Reply:
Correct. [This comment](https://issues.apache.org/jira/browse/SOLR-7896?focusedCommentId=16643457&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16643457) from the link i posted earlier actually shows what the login page would look like. You can also create a test deployment using the `solr-v1554255185081` image in our test project, if you'd want to see it in action yourself.

Reply:
LGTM.